### ISSUE TITLE
Calls to Step Functions return 500

### ISSUE THREAD

**Issue by mmanes** [localstack/helm-charts#5]
Env: Docker for Desktop 3.0.3 on macOS 10.15.7

```
$ aws --endpoint-url http://localhost:4566 stepfunctions list-state-machines
An error occurred (500) when calling the ListStateMachines operation (reached max retries: 2):
```

Relevant log entry:
```requests.exceptions.ConnectionError: MyHTTPConnectionPool(host='localstack-78c5774d85-gcrkn', port=8083): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7faaf9712730>: Failed to establish a new connection: [Errno 111] Connection refused'))```

It appears the Step Functions proxy is trying to forward to port 8083, which is not the port the service is listening on.

**Comment by mmanes** [comment]
After running with debug enabled, I see this in the logs:

```
2020-12-23 20:27:29.408: Configure [PORT] to [tcp://10.107.185.120:4566]
2020-12-23 20:27:29.418: Configure [Account] to [000000000000]
2020-12-23 20:27:29.418: Configure [Region] to [us-east-1]
2020-12-23 20:27:29.419: Configure [Lambda Endpoint] to [http://localstack-77675bf4cf-jlckm:4566]
2020-12-23 20:27:29.419: Configure [DynamoDB Endpoint] to [http://localstack-77675bf4cf-jlckm:4566]
2020-12-23 20:27:29.419: Configure [SQS Endpoint] to [http://localstack-77675bf4cf-jlckm:4566]
2020-12-23 20:27:29.419: Configure [SNS Endpoint] to [http://localstack-77675bf4cf-jlckm:4566]
2020-12-23 20:27:29.420: Configure [Step Functions Endpoint] to [http://localstack-77675bf4cf-jlckm:4566]
2020-12-23 20:27:29.428: Loaded credentials from environment
Exception in thread "main" java.lang.NumberFormatException: For input string: "tcp://10.107.185.120:4566"
at java.base/java.lang.NumberFormatException.forInputString(Unknown Source)
at java.base/java.lang.Integer.parseInt(Unknown Source)
at java.base/java.lang.Integer.<init>(Unknown Source)
at com.amazonaws.stepfunctions.local.runtime.config.ConfigOption.getIntegerValue(ConfigOption.java:81)
at com.amazonaws.stepfunctions.local.runtime.Config.getPort(Config.java:73)
at com.amazonaws.stepfunctions.local.StepFunctionsLocal.start(StepFunctionsLocal.java:72)
at com.amazonaws.stepfunctions.local.StepFunctionsLocal.start(StepFunctionsLocal.java:65)
at com.amazonaws.stepfunctions.local.Launcher.main(Launcher.java:15)
```

It seems the stepfunctions service is not starting due to it expecting the PORT environment variable to be an integer port instead of `tcp://10.107.185.120:4566`. Modifying `stepfunctions_starter.py` to overwrite that variable allowed the service to start.

```
cmd = ('cd %s; PORT=8083 java -Dcom.amazonaws.sdk.disableCertChecking -Xmx%s -jar StepFunctionsLocal.jar '
           '--dynamodb-endpoint %s --sns-endpoint %s '
           '--sqs-endpoint %s --aws-region %s --aws-account %s --step-functions-endpoint %s') % (
        install.INSTALL_DIR_STEPFUNCTIONS, MAX_HEAP_SIZE, dynamodb_endpoint,
        sns_endpoint, sqs_endpoint, aws_stack.get_region(), TEST_AWS_ACCOUNT_ID, sfn_endpoint)
```
I'm certain there is a better fix, but since port 8083 is already hardcoded later in the file, it may be acceptable.

**Comment by rosscdh-tpg** [comment]
same issue today

```
localstack_main  | Error opening zip file or JAR manifest missing : aspectjweaver-1.9.7.jar
```

this is in localstack latest

```
localstack/localstack          latest    a92aac1f292a   6 hours ago    1.08GB
```

**Comment by rosscdh-tpg** [comment]
looks to be there are 2 issues
1. StepFunctionsLocal.jar is out of date 
2. the aspectjweaver-1.9.7.jar is not present in the image

**Comment by alexrashed** [comment]
Hi @rosscdh-tpg.
You are commenting on a ticket which has been closed 2.5 years ago.
Your issue also doesn't really seem to be related to the Helm Chart of LocalStack. If you are not using the Helm chart, please make sure to create an issue in https://github.com/localstack/localstack instead and try to give as much information as possible (the best case would be a reproducible sample).
Thanks!
